src/include/catalog/postgres/pg_language_impl.h

```diff
@@ -0,0 +1,99 @@
+#pragma once
+
+#include <string>
+
+#include "catalog/postgres/pg_language.h"
+#include "common/managed_pointer.h"
+#include "storage/projected_row.h"
+#include "storage/storage_defs.h"
+
+namespace noisepage::storage {
+class RecoveryManager;
+class SqlTable;
+
+namespace index {
+class Index;
+}  // namespace index
+}  // namespace noisepage::storage
+
+namespace noisepage::transaction {
+class TransactionContext;
+}  // namespace noisepage::transaction
+
+namespace noisepage::catalog::postgres {
+class Builder;
+
+/** The NoisePage version of pg_language. */
+class PgLanguageImpl {
+ public:
+  /**
+   * Prepare to create pg_language.
+   * Does NOT create anything until the relevant bootstrap functions are called.
+   *
+   * @param db_oid          The OID of the database that pg_language should be created in.
+   */
+  explicit PgLanguageImpl(db_oid_t db_oid);
+
+  /** Bootstrap the projected row initializers for pg_language. */
+  void BootstrapPRIs();
```

Comment:
This should probably be private and called by `PgLanguageImpl::Bootstrap()`.

Reply:
Per previous conversation, no can do right now